packages/features/bookings/Booker/types.ts

```diff
@@ -75,6 +86,30 @@ export interface BookerProps {
   isInstantMeeting?: boolean;
 }
 
+export type WrappedBookerProps = {
```

Comment:
Can we merge `BookerProps` and `WrappedBookerProps` into `BookerProps` ? I see we use `BookerProps & WrappedBookerProps` and I think they can be merged because the "dumb" component needs to have all of these props.